sdk/spring/spring-reference.yml

```diff
@@ -51,6 +51,46 @@
                 groupId: com.microsoft.azure                                                        # The associated group-id of the target Azure Spring library version.
                 artifactId: azure-spring-boot-bom                                                   # The associated artifact-id of the target Azure Spring library version.
                 version: 2.2.4                                                                      # The target version of Azure Spring library mapped with Spring Boot version range.
+- name: Azure Spring Cloud Dependencies
+  content:
+    - name: Azure Spring Cloud Dependencies
+      description: BOM for Microsoft Azure Spring Cloud support.
+      artifacts:
+        - artifactId: azure-spring-cloud-dependencies
+          groupId: com.azure.spring
+          versionGA: 2.4.0
+          description: BOM for Microsoft Azure Spring Cloud support.
+          type: spring
+          links:
+            repopath: https://search.maven.org/artifact/com.azure.spring/azure-spring-cloud-dependencies
+            github: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/boms/azure-spring-cloud-dependencies
+          dependencyPattern:
+            maven: |-
+              <dependencyManagement>
+                <dependencies>
+                  <dependency>
+                    <groupId>com.azure.spring</groupId>
+                    <artifactId>azure-spring-cloud-dependencies</artifactId>
+                    <version>${azure.version}</version>
+                    <type>pom</type>
+                    <scope>import</scope>
+                  </dependency>
+                </dependencies>
+              </dependencyManagement>
+            gradle: |-
+              dependencyManagement {
+                imports {
+                  mavenBom "com.azure.spring:azure-spring-cloud-dependencies:${azure.version}"
+                }
+              }
+          springProperties:
+            starter: false
+            compatibilityRange: "[2.2.11.RELEASE,2.4.0-M1)"
+            mappings:
+              - compatibilityRange: "[2.0.0.RELEASE,2.3.0.M1)"
+                groupId: com.microsoft.azure
+                artifactId: spring-cloud-azure-dependencies
```

Comment:
what's the difference between spring-cloud-azure-dependencies and azure-spring-cloud-dependencies?

Reply:
The mapping property is used for version mapping with Spring Boot. So it's necessary for us spring libraries as our libs cannot support all Spring Boot versions, I think.